fencing/fence_dummy

```diff
@@ -16,14 +16,18 @@ import time
 import random
 import atexit
 import getopt
+import subprocess
+import shlex
 
 AGENT_VERSION = "4.0.0"
 OCF_VERSION = "1.0"
 SHORT_DESC = "Dummy fence agent"
 LONG_DESC = """fence_dummy is a fake fencing agent which reports success
-based on its mode (pass|fail|random) without doing anything."""
+based on its mode (pass|fail|random|ping) without doing any actual fencing."""
+verbose = 0
+PING_BINARY = "/usr/bin/ping"
```

Comment:
There's this long standing issue with `ping` vs. `ping6` schizophrenia
that hadn't been solved in some distros until recently:
https://fedoraproject.org/wiki/Changes/PingIpv6
and some require the dichotomy till today.